guides/common/modules/ref_best-practices-for-errata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-errata_{context}"]
+= Best practices for errata
+
+* {Project} shows which errata are applicable for each content host by comparing package lists with installed and available packages.
+* Use errata to add patches for security issues to a frozen set of content without unnecessarily updating other unaffected packages.
+* View errata on the content hosts page and choose between errata from the current content view and lifecycle environment and the _Library_ lifecycle environment which contains the latest synchronized packages.
```

Comment:
```suggestion
* View errata on the content hosts page and choose between errata from the current content view and lifecycle environment and the _Library_ lifecycle environment, which contains the latest synchronized packages.
```
I'm not sure what "choose between" means. Do you mean "compare"?

Reply:
Yes, changed to "compare".